### PR TITLE
add scovie to app catalog

### PR DESCRIPTION
**Pull request by eldertek**
Please add Scovie, an open-source digital signage system for high schools, to the YunoHost Apps Catalog.

### CODE REVIEW

File: apps.toml [hunk 1]
```diff
@@ -3095,6 +3095,12 @@ state = "notworking"
 subtags = [ "forge" ]
 url = "https://github.com/drfred1981/scm-manager_ynh"
 
+[scovie]
+category = "publishing"
+state = "working"
+subtags = [ "websites" ]
+url = "https://github.com/eldertek/scovie_ynh"
```
Comment (ericgaspar) [inline apps.toml:3102]: it will be better if you can transfer the app to the YunoHost app repo

Comment (eldertek) [reply apps.toml]: Changed to yunohost-apps, but cannot transfer ownership : "You don’t have the permission to create public repositories on YunoHost-Apps"

Comment (ericgaspar) [reply apps.toml]: invite send

Comment (eldertek) [reply apps.toml]: I imported the repository !

**Comment by ericgaspar** [comment]
Thanks.
I know this is very tempting to put variables in `_common.sh` file but don't. This will make transition to package v2 harder. It is better to apply the example_ynh template as close as you can.

**Comment by eldertek** [comment]
There is problem with readme generator : 
Official app website: https://github.com/eldertek/scovie
Upstream app code repository: https://github.com/eldertek/scovie_ynh
YunoHost documentation for this app: https://yunohost.org/app_scovie_ynh
Report a bug: https://github.com/YunoHost-Apps/scovie_ynh_ynh/issues

**Comment by ericgaspar** [comment]
The app id is wrong `"id": "scovie_ynh",`
Should be `"id": "scovie",`

**Comment by eldertek** [comment]
Can I also edit  ?
```
These projects used scovie_ynh:

https://github.com/eldertek/scovie_ynh
```

**Review by ericgaspar — APPROVED** [review]

**Comment by ericgaspar** [comment]
It might be better to decouple your app from the yunohost app script:
- develop your app in your repo: https://github.com/eldertek/scovie
- deploy your application with https://github.com/yunohost/scovie_ynh

**Comment by eldertek** [comment]
I wanted to edit like this : 

These projects used scovie_ynh:

https://github.com/YunoHost-Apps/scovie_ynh

**Comment by ericgaspar** [comment]
What do you mean?

**Comment by eldertek** [comment]
I want to delete the https://github.com/eldertek/scovie_ynh on my account because right now it exists at Yunohost-Apps, but in my readme it continue to appear as this url "https://github.com/eldertek/scovie_ynh" and not "https://github.com/Yunohost-Apps/scovie_ynh"

Here : https://github.com/YunoHost-Apps/scovie_ynh/blob/testing/README.md#links

**Comment by ericgaspar** [comment]
> It might be better to decouple your app from the yunohost app script:
> 
>     * develop your app in your repo: https://github.com/eldertek/scovie
> 
>     * deploy your application with https://github.com/yunohost/scovie_ynh

You should really keep you app separated (in your repo) from the yunohost script!

**Comment by eldertek** [comment]
I don't understand my app is at scovie, not scovie_ynh

scovie: https://github.com/eldertek/scovie 

Je parle aussi français, et j'ai du mal à comprendre, le code de mon application est sur https://github.com/eldertek/scovie et l'app yunohost est bien sur https://github.com/yunohost/scovie_ynh 

Elles sont bien séparés ?

**Pull request merged**